streams/src/test/java/org/apache/kafka/streams/integration/EosIntegrationTest.java

```diff
@@ -442,8 +442,8 @@ public void shouldNotViolateEosIfOneTaskFails(final String eosConfig, final bool
             writeInputData(dataAfterFailure);
 
             waitForCondition(
-                () -> uncaughtException != null, MAX_WAIT_TIME_MS,
-                "Should receive uncaught exception from one StreamThread.");
+                    () -> uncaughtException != null, MAX_WAIT_TIME_MS,
+                    "Should receive uncaught exception from one StreamThread.");
```

Comment:
nit: avoid unnecessary reformatting (maybe need to change your IDE setting to disable auto-reformatting)